.travis.yml

```diff
@@ -1,6 +1,11 @@
-sudo: required
+sudo: false
 dist: trusty
 language: generic
+services:
+  - docker
+cache:
```

Comment:
It looks like setting:

`cache: ccache`

Will automatically install/add to path ccache, and cache the correct directory.

https://docs.travis-ci.com/user/caching/

Reply:
EDIT: ah but perhaps it won't pass that to the docker container. So can just change:

```
cache:
  directories:
    - $HOME/.ccache
```

to `cache: ccache`

Reply:
Ok, I updated.